jq/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM archlinux/base
```

Comment:
So far we use a few base images to save space (and shorten pull time) when testing several implementations. @kanaka - do we still keep this practice?

I'm sure there's a `jq` package in Ubuntu as well.

Reply:
That's a fair point.

Though the vivid LTS has a lot of outdated packages (the jq in ubuntu's repositories is ver 1.5, I think that one tends to segfault for some reason)

I'll try to use the same base image, let's see how that goes

Reply:
vivid is pretty old; use `FROM ubuntu:18.04` (bionic - the latest LTS release) as newer implementations do.

Reply:
Ended up using ubuntu:bionic (zig also uses this, so theoretically, no loss?)

Reply:
Looks OK. I guess you could use curl instead of installing wget, but not sure it matters. Anyway need to wait for @kanaka to build this image and push it to docker-hub so the tests in Travis can run; we'll hear his opinion.